easybuild/easyconfigs/g/goolf/goolf-1.5.16.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "Toolchain"
+
+name = 'goolf'
+version = '1.5.16'
```

Comment:
Is that necessary? I would have to recompile all components and we already have GDAL software depend on this toolchain (in production), so recompile GDAL as well and ask customer to use different version. 

Let me know, I have prepared GDAL-1.9.2-goolf-1.5.16.eb to push.